### ISSUE TITLE
[BITV] 9.2.4.7/8.1 - Button "Settings" - Since the focus highlighting is solely accomplished through a color change, the contrast ratio between the focused and unfocused state should be at least 3:1, but currently it is only 1.1:1. (1)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/server#37017]
- [ ] Fix
- [ ] Backport

<img width="312" alt="4d6a46885324ac5f2c34dd43dde155f3" src="https://user-images.githubusercontent.com/1315170/222543685-7ae5342b-8484-4873-8672-449a4b65f491.png">

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-91f2646afa-v8-n1

</details>
